1ES-Provision.yml

```diff
@@ -0,0 +1,126 @@
+trigger:
+  batch: none
+  branches:
+    include:
+    - non-exist
+pr:
+- non-exist
+
+variables:
+- group: devresourcekeyvault
+- name: dotnetVersion
+  value: 5.0.200
+- name: runCodesignValidationInjection
+  value: false
+- name: NugetSecurityAnalysisWarningLevel
+  value: none
+- name: repos
+  value:
+    https://github.com/MicrosoftDocs/azure-docs-pr --branch master --profile --timeout 100 --regression-rules
+    https://github.com/MicrosoftDocs/sql-docs-pr --branch master --profile --timeout 120 --regression-rules
+    https://github.com/dotnet/docs --branch main --timeout 60 --regression-rules
+    https://github.com/MicrosoftDocs/learn-pr --branch master --timeout 100 --no-dry-sync --regression-rules
+    https://github.com/MicrosoftDocs/windowsserverdocs-pr --branch master  --timeout 45 --regression-rules
+    https://github.com/MicrosoftDocs/VBA-Docs --branch master --timeout 60 --regression-rules
+    https://github.com/MicrosoftDocs/azure-devops-docs-pr --branch master --timeout 30 --regression-rules
+    https://github.com/dotnet/AspNetCore.Docs --branch main --profile --timeout 30 --output-html
+    https://github.com/MicrosoftDocs/microsoft-365-docs-pr.zh-CN --branch live-sxs --profile --timeout 25
+    https://github.com/MicrosoftDocs/powerbi-docs-pr.de-DE --branch live-sxs --timeout 25
+    https://github.com/MicrosoftDocs/PowerShell-Docs --timeout 75 --branch staging
+    https://github.com/dotnet/roslyn-api-docs --branch live --profile --timeout 105
+    https://github.com/Azure/azure-docs-rest-apis --branch master --timeout 130 --error-level Warning
+    https://github.com/MicrosoftDocs/dynamics365-docs-odata-apis --branch master --timeout 20
+    https://github.com/MicrosoftDocs/mc-docs-pr --branch master --timeout 70
+    https://github.com/MicrosoftDocs/dynamics365smb-devitpro --branch master --timeout 15
+    https://github.com/MicrosoftDocs/DevSandbox --branch master --timeout 30
+    https://github.com/MicrosoftDocs/test --branch master --timeout 20 --no-dry-sync
+    https://github.com/MicrosoftDocs/DocsRoot --branch master --timeout 20
+    https://dev.azure.com/cpubwin/windows-compatibility/_git/windows-compatibility --branch main --timeout 20
+    https://github.com/MicrosoftDocs/azure-docs-cli --branch master --timeout 50
+    https://github.com/MicrosoftDocs/dataprep-dotnet-pr --branch master --timeout 30
+    https://github.com/MicrosoftDocs/azure-mediaplayer-typescript --branch master --timeout 30
+    https://github.com/MicrosoftDocs/quantum-docs-private --timeout 30 --branch main
+    https://github.com/microsoftgraph/microsoft-graph-docs --branch main --timeout 30
+    https://github.com/MicrosoftDocs/bot-docs-pr --branch master --timeout 40 --no-dry-sync --dry-run --public-template
+    https://github.com/MicrosoftDocs/azure-docs-pr --branch master --profile --timeout 50 --dry-run --regression-rules
```

Comment:
Can the parameters irrelevant to WarmUp be removed?

Reply:
ok, will remove